### DISCUSSION TITLE
Issue while redeploying the Cert-pre-processer-Flinks Jobs.

### DISCUSSION THREAD

**Discussion by kiranby** [Sunbird-Knowlg/Community#113] [Bugs]
Hi Team,

While redeploying the **Cert-pre-processer-Flinks Jobs** of **release-5.1.0** with the tag of **release-5.2.0_RC8**,  job gets failed with below error.

**STDERR:
Error: UPGRADE FAILED: YAML parse error on datapipeline_jobs/templates/flink_job_configmap.yaml: error converting YAML to JSON: yaml: line 63: could not find expected ':'
MSG:  non-zero return code**

Thanks..

**Comment by kiranby** [comment]
@Team, can you please guide us to resolve the issue, thanks.

**Reply by pallakartheekreddy** [reply]
@kiranbysani Cert-pre-processer-Flinks Jobs is not belongs to Knowlg building block. Please post this question in Lern discussion forum. 
https://github.com/Sunbird-Lern/Community/discussions
@AmiableAnil @Jayaprakash8887
